packages/components/src/components/SquiggleViewer/getSqValueWidget.tsx

```diff
@@ -155,11 +158,13 @@ export function getSqValueWidget(value: SqValueWithContext): ValueWidget {
     case "Calculator": {
       return {
         render: (settings) => (
-          <Calculator
-            valueWithContext={value}
-            environment={environment}
-            settings={settings}
-          />
+          <CalculatorSampleCountValidation calculator={value}>
```

Comment:
I wasn't very sure about this part. It seemed important to wrap the entire Calculator with the Validation component, because it contains hooks that would start running the calculator function, and we wanted to avoid those.

I could also just move this all to Calculator, and rename the current <Calculator> component to <CalculatorInner> or something.

Reply:
Yeah, having this in Calculator would be better:
- `<Calculator>` = `<CalculatorSampleCountValidation><InnerCalculator /></CalculatorSampleCountValidation>`
- `InnerCalculator` = current implementation of `Calculator`

`getSqValueWidget` doesn't need to know implementation details.